cdk/lib/gatehouse.ts

```diff
@@ -535,7 +535,7 @@ export class Gatehouse extends GuStack {
 			},
 			computeConfig: {
 				minCapacityUnits: 1,
-				maxCapacityUnits: 16,
+				maxCapacityUnits: 8,
```

Comment:
this was the wrong setting, it wasn't reverting the change in the db capacity but the DMS config